src/js/components/AppListComponent.jsx

```diff
@@ -433,16 +433,31 @@ var AppListComponent = React.createClass({
   },
 
   getInlineDialog: function (appNodes = []) {
-    var state = this.state;
+    var {props, state} = this;
```

Comment:
Regarding `apps` and `fetchState` I really want to keep the `state.` prefix as it signals us we shouldn't modify those values as we're dealing with a piece of state.
The other suggestion makes sense. Thanks!